nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html

```diff
@@ -53,6 +55,13 @@
               status: {
                   type: Object,
                   notify: true,
+              },
+
+              isAiEnabled: {
```

Comment:
I would call it "isInsightEnabled" as you could have 3rd party AI providers working.

Reply:
@tiagocardoso thanks! I'll go with Nelson suggestion of not using a property and directly using a function instead.